gamePlay.js

```diff
@@ -0,0 +1,73 @@
+import { displayRevealedCell, revealWholeGrid, resetDisplay} from "./display.js";
+
+
+export const changeDisplayedCell = function (cell, row, col, gridObject, game) {
+    let currentCell = gridObject.getCell(row, col);
+    console.log(gridObject.getCell(row, col));
+    if (game.gameOver){
+        return;
+    }
+
+    if (game.flaggingOn){
+        let cellObject = gridObject.getCell(row, col);
+        toggleFlagging(cellObject, cell);
+        return;
+    }
+
+    if (currentCell.numMinesAround == 0){
+        gridObject.changeCellProperty(row, col, "isRevealed", true);
+        const revealedCells = gridObject.revealZeros(row, col);
+        revealedCells.forEach(c =>{
+            displayRevealedCell(c);
+        } );
```

Comment:
if it's a single line you don't need `{ ... }`
```suggestion
        revealedCells.forEach(c => displayRevealedCell(c));
```

Reply:
makes sense